keyboards/ambergon/multipleglove/readme.md

```diff
@@ -0,0 +1,36 @@
+# MultipleGlove
+
+![MultipleGlove](https://github.com/ambergon/MultipeGlove/blob/master/img/1.jpg)
+![MultipleGlove](https://github.com/ambergon/MultipeGlove/blob/master/img/2.jpg)
+![MultipleGlove](https://github.com/ambergon/MultipeGlove/blob/master/img/6.jpg)
```

Comment:
These are all dead links. These images should be hosted on Imgur if you'd like them to be included here.